src/components/Swap/SwapActionButton/SwapButton.tsx

```diff
@@ -90,9 +94,14 @@ export default function SwapButton({ disabled }: { disabled: boolean }) {
       // Only close the review modal if the swap submitted (ie no-throw).
       setOpen(false)
     } catch (e) {
-      console.error(e) // ignore error
+      throwAsync(
+        new SwapError({
+          header: 'Swap Failed',
```

Comment:
I think this should read "Swap Error", because the swap did not necessarily fail. Specifically, see https://github.com/Uniswap/widgets/blob/main/src/hooks/useUniversalRouter.ts#L62-L66.